lib/events/filesessions/fileasync_test.go

```diff
@@ -849,8 +942,14 @@ func newUploaderPack(ctx context.Context, t *testing.T, cfg uploaderPackConfig)
 	ctx, cancel := context.WithCancel(ctx)
 	t.Cleanup(cancel)
 
+	clock := cfg.clock
+	var fakeClock *clockwork.FakeClock
```

Comment:
Why do we need two clocks?

Reply:
We don't actually, this is way more confusing than just updating the existing tests to type assert the fake clock. I needed to be able to use a `clockwork.RealClock` since `synctest` doesn't play nice with `clockwork.FakeClock`